examples/docs/es/dialog.md

```diff
@@ -272,24 +276,25 @@ Dialog's content can be centered.
 :::
 
 :::tip
-`center` only affects Dialog's header and footer. The body of Dialog can be anything, so sometimes it may not look good when centered. You need to write some CSS if you wish to center the body as well.
+`center` afecta a cabezazos y pies de Diálogo.El cuerpo de Diálogo puede ser cualquier cosa,asi que a veces no puede verse bien cuando está centrado.Necesitas escribir algunos CSS si quieres centrar el cuerpo también.
 :::
 
 :::tip
-If the variable bound to `visible` is managed in Vuex store, the `.sync` can not work properly. In this case, please remove the `.sync` modifier, listen to `open` and `close` events of Dialog, and commit Vuex mutations to update the value of that variable in the event handlers.
+Si el variable obligado a `visible` es manejado en el almacén Vuex,el `.sync` no puede funcionar correctamente.En este caso,quita el modificador `.sync` por favor,escucha a eventos `open` y `close` de Diálogo,y comete Vuex mutaciones para actualizar el valor de la variable en los controladores de eventos.
+
 :::
 
-### Attributes
+### Atributo
 
-| Attribute      | Description          | Type      | Accepted Values       | Default  |
+| Atributo      | Descripción          | Tipo      | Valores aceptados       | Defecto  |
 |---------- |-------------- |---------- |--------------------------------  |-------- |
-| visible   | visibility of Dialog, supports the .sync modifier | boolean | — | false |
-| title     | title of Dialog. Can also be passed with a named slot (see the following table) | string    | — | — |
-| width     | width of Dialog | string    | — | 50% |
-| fullscreen     | whether the Dialog takes up full screen | boolean    | — | false |
+| visible   | visibilidad del Diálogo, apoya el modificador .sync | boolean | — | false |
+| title     | título de Diálogo. También se puede pasar con una ranura nombrada (ver la tabla siguiente) | string    | — | — |
+| width     | anchura de Diálogo| string    | — | 50% |
+| fullscreen | si el diálogo ocupa pantalla completa | boolean     | — | false |
 | top      | value for `margin-top` of Dialog CSS | string    | — | 15vh |
-| modal     | whether a mask is displayed | boolean   | — | true |
-| modal-append-to-body     | whether to append modal to body element. If false, the modal will be appended to Dialog's parent element | boolean   | — | true |
+| modal     | si se muestra una máscara | boolean   | — | true |
+| modal-append-to-body | si adjuntar modal al elemento de cuerpo. Si es falso,el modal se agregará al elemento principal de Diálogo | boolean   | — | true |
 | append-to-body     | whether to append Dialog itself to body. A nested Dialog should have this attribute set to `true` | boolean   | — | false |
```

Comment:
There's more to translate from this line.

Reply:
I see,today I will finish the translation of lines left